templates/README.md

```diff
@@ -73,6 +73,28 @@ Please see [Link][zero-k8s-guide]
 - AWS IAM integration with Kubernetes RBAC
 ...
 
+#### Sending Email with Sendgrid
+Setup: If you initialized your infrastructure with a sendgridApiKey, you should have a verified domain with Sendgrid once your infrastructure is setup. We should have created for you 3 route53 entries, and [verified you domain with sendgrid][sendgrid-domain-verification]
```

Comment:
Should be "We should have **created 3 route53 entries for you** and verified **your** domain with sendgrid"